docs/reference/ccr/apis/auto-follow/get-auto-follow-pattern.asciidoc

```diff
@@ -64,6 +64,12 @@ GET /_ccr/auto_follow/<auto_follow_pattern_name>
   retrieve; if you do not specify a name, the API returns information for all
   collections
 
+//==== Authorization
+
+//TBD
```

Comment:
This is the same as put and delete auto follow pattern APIs.

Reply:
Done, thanks!